CHANGELOG.md

```diff
@@ -1,9 +0,0 @@
-# Change Log
-
-All notable changes to the "spectral" extension will be documented in this file.
```

Comment:
Any reason not to keep changelog?

Reply:
Semantic Release will populate release notes automatically making this a bit redundant. 

For Spectral and big projects we maintain develop branches and put in extra effort on CHANGELOGs but for little libs and things like this we want zero effort git-flow-release.